chromadb/config.py

```diff
@@ -128,6 +128,8 @@ def chroma_server_auth_credentials_file_non_empty_file_exists(
         "/api/v1": ["GET"],
         "/api/v1/heartbeat": ["GET"],
         "/api/v1/version": ["GET"],
+        "/docs": ["GET"],
```

Comment:
Let's remove this for now since there is a way to get it locally and I don't think this is well - maintained

Reply:
Done